Jenkinsfile

```diff
@@ -10,7 +10,7 @@ pipeline {
   stages {
     stage('Install packages') {
       steps {
-        sh 'npm install'
+        sh 'npm install -f'
```

Comment:
IMO the CI related changes to redownload packages should be dealt with in a separate issue / pr.

Reply:
I've already created an issue for that: https://github.com/OpenPaaS-Suite/esn/issues/36. For this PR I need to do it here or I'll have to create another PR...